Source/WebCore/crypto/mac/CryptoKeyECMac.cpp

```diff
@@ -252,16 +251,27 @@ RefPtr<CryptoKeyEC> CryptoKeyEC::platformImportSpki(CryptoAlgorithmIdentifier id
     index += oidSize + 1; // Read named curve OID, BIT STRING
     if (keyData.size() < index + 1)
         return nullptr;
-    index += bytesUsedToEncodedLength(keyData[index]) + 1; // Read length, InitialOctet
-
-    if (!doesUncompressedPointMatchNamedCurve(curve, keyData.size() - index))
-        return nullptr;
-
+    index += bytesUsedToEncodedLength(keyData[index]); // Read length
+    uint8_t initialOctet = keyData[index];
+    index += 1;
+        
+        
+    bool uncompressed = initialOctet == 4 or initialOctet == 04 ;
+    bool compressed = initialOctet == 03 or initialOctet == 02;
+        
+        
     CCECCryptorRef ccPublicKey = nullptr;
-    if (CCECCryptorImportKey(kCCImportKeyBinary, keyData.data() + index, keyData.size() - index, ccECKeyPublic, &ccPublicKey))
-        return nullptr;
-
-    return create(identifier, curve, CryptoKeyType::Public, PlatformECKeyContainer(ccPublicKey), extractable, usages);
+        
+    if ( uncompressed ){
```

Comment:
Coding style probably tells us to write it as `if (uncompressed) {`

Reply:
I changed  LayoutTests/platform/glib/TestExpectations as the fix was only made in CryptoKeyECMac